docs/admin/kubeadm.md

```diff
@@ -13,188 +13,245 @@ Running `kubeadm init` bootstraps a Kubernetes cluster. This consists of the
 following steps:
 
 1. kubeadm runs a series of pre-flight checks to validate the system state
-before making changes. Some checks only trigger warnings, others are
-considered errors and will exit kubeadm until the problem is corrected or
-the user specifies `--skip-preflight-checks`.
+   before making changes. Some checks only trigger warnings, others are
+   considered errors and will exit kubeadm until the problem is corrected or the
+   user specifies `--skip-preflight-checks`.
 
 1. kubeadm generates a token that additional nodes can use to register
-themselves with the master in future.  Optionally, the user can provide a token.
+   themselves with the master in future.  Optionally, the user can provide a
+   token.
 
-1. kubeadm generates a self-signed CA using openssl to provision identities
-for each node in the cluster, and for the API server to secure communication
-with clients.
+1. kubeadm generates a self-signed CA using openssl to provision identities for
```

Comment:
It's not using openssl and it's generating much more than only a CA and a serving cert